ci-operator/jobs/infra-periodics.yaml

```diff
@@ -158,6 +158,7 @@ periodics:
         --query=repo:openshift/origin
         repo:operator-framework/operator-sdk
         repo:tnozicka/openshift-acme
+        repo:openshift/openshift-azure
```

Comment:
I think we also need to add openshift-azure to the -rotten and -close jobs.

Reply:
That's fair, I didn't add it thinking we didn't want to automate that but I can push the commit to add it to the other two jobs 👍

Reply:
Without all three jobs, this only achieves one portion of the closing out stale issues process :)